cli/config/configuration-nemu.toml.in

```diff
@@ -0,0 +1,331 @@
+# Copyright (c) 2017-2019 Intel Corporation
```

Comment:
Since 2017~2018 are passed, I think this should be 2019?

Reply:
If we are inheriting things from files from the past, it is normal to have a year range covering all the years bits came from